models/balancer/arbitrum/balancer_v2_arbitrum_trades.sql

```diff
@@ -72,13 +73,13 @@ WITH
         FROM
             dexs
             LEFT JOIN {{ ref('balancer_v2_arbitrum_bpt_prices') }} bpt_prices
-                ON bpt_prices.contract_address = dexs.token_bought_address
+                ON bpt_prices.contract_address = CAST(dexs.token_bought_address AS VARCHAR)
```

Comment:
i assume you'll update this back to varbinary, once you do `bpt_prices`?

Reply:
i'll only comment here, but applies universally

Reply:
Correct. Will remove that in a PR where returned results in `bpt_prices` are the proper data type.